### PR TITLE
Add csi-driver-nfs image

### PR DESCRIPTION
**Pull request by Fedosin**

### CODE REVIEW

File: ci-operator/config/openshift/csi-driver-nfs/openshift-csi-driver-nfs-master.yaml [hunk 1]
```diff
@@ -1,17 +1,26 @@
...
-    tag: golang-1.11
+    tag: golang-1.13
 canonical_go_repository: github.com/kubernetes-csi/csi-driver-nfs
+images:
+- dockerfile_path: images/Dockerfile.openshift.ci
+  from: base
+  inputs:
+    root:
+      as:
+      - "0"
```
Comment (smarterclayton) [inline ci-operator/config/openshift/csi-driver-nfs/openshift-csi-driver-nfs-master.yaml]: This is generally incorrect, why are you doing this?  Note that your Dockerfile that you build here should be your official one (the one used in OSBS), not a CI only image.

If the other PR had this issue, we need to fix that too.

Comment (Fedosin) [reply ci-operator/config/openshift/csi-driver-nfs/openshift-csi-driver-nfs-master.yaml]: It comes from here: https://github.com/openshift/release/blob/master/ci-operator/config/openshift/cloud-provider-openstack/openshift-cloud-provider-openstack-release-4.1.yaml#L14-L22
I just copied the image definition. Should I remove the "inputs"?

Comment (Fedosin) [reply ci-operator/config/openshift/csi-driver-nfs/openshift-csi-driver-nfs-master.yaml]: Done :)

**Comment by petr-muller** [comment]
/uncc 

people from https://github.com/openshift/release/blob/master/ci-operator/config/openshift/csi-driver-nfs/OWNERS should review this

**Pull request merged**